root/etc/s6-overlay/s6-rc.d/init-flexget-config/run

```diff
@@ -5,17 +5,19 @@ mkdir -p \
     /config/.flexget \
     /data
 
+FG_CONFIG_FILE="${FG_CONFIG_FILE:-/config/.flexget/config.yml}"
+FG_LOCK_FILE="${FG_CONFIG_FILE/config.yml/.config-lock}"
```

Comment:
There's an edge case here if the user provides a `yaml` file rather than a `yml` file but I'm not *too* concerned about it.

Reply:
All the docs only refer to yml files. Should we handle yaml extensions?

Reply:
I don't think it's critical, but it is a valid extension for YAML files, so people are more likely to use it than some other random extension that we definitely wouldn't consider supporting.